packages/flutter_tools/lib/executable.dart

```diff
@@ -5,6 +5,11 @@
 import 'dart:async';
 
 import 'runner.dart' as runner;
+import 'src/base/context.dart';
+// The build_runner code generation is provided here to make it easier to
+// avoid introducing the dependency into google3.
```

Comment:
nit: This aspect might be useful to others companies, so I'd phrase it as "avoid introducing the dependency into build systems that use their own codegen methods, such as Google's internal infrastructure".

Reply:
This is only an issue for google3 because the build packages are not synced. In theory it could run...